doc/guides/getting-started/finding_modules.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Finding and Using Modules
+description: This tutorial will explain how to use modules in RIOT.
+---
+
+import Contact from '@components/contact.astro';
+
+A core concept in RIOT is the use of modules.
+Modules are a way to organize code and data in a way that makes it easy to reuse and share.
+In the previous tutorial, we created a simple hello world program and you might have noticed that we included a module called `ztimer`.
+`ztimer` is one of the core modules in RIOT and provides a way to work with timers,l both software and hardware timers are supported.
+
+Lets take a look at how we can use modules in RIOT and where to find them.
+
+## Step 1: Finding modules
+
+RIOT provides a number of modules that you can use in your application, including modules for networking, sensors, and more.
+Depending on the goal of your application, you might need to use different modules and sometimes even create your own. So where can you find these modules?
+
+There are several approaches to this and none of them are wrong.
+
+### Approach 1: The API Documentation
+
+![API Documentation Website](img/finding_modules/01_api_doc.png)
+
+The first approach is to look at the API documentation. The API documentation provides a list of all modules and functions that are available in RIOT and can be found [here](https://doc.riot-os.org/topics.html).
+
+It can be a bit overwhelming to find the right module in the API documentation, but it is a good starting point if you know what you are looking for.
+
+### Approach 2: The Examples
+
+![RIOT Examples](img/finding_modules/02_examples.png)
```

Comment:
I just realized two things:

1. We do not explain how to work with images in the guides, in the "how to guide"-guide. We should fix that.
2. The images do not show in the rendering on GitHub. Do you think there is a way to fix that? Or is that a shortcoming of githubs PR preview?

Reply:
Okay, for 2. this is a shortcoming in GitHubs PR preview. 🤡 it shows correct in the branch: https://github.com/AnnsAnns/RIOT/blob/tutorials/getting_started/doc/guides/getting-started/finding_modules.mdx

Reply:
This screenshot is very outdated.